Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM alpine:latest
+
+# Add the dependencies
+RUN apk add bash curl wget unzip libc6-compat libstdc++
+
+# Get the latest release of sloctl
+RUN curl -s https://api.github.com/repos/nobl9/sloctl/releases/latest | \
```

Comment:
I'm on the fence about this.  on the one hand, sloctl shouldn't change all that much and be backwards compatible so it should be ok to use the latest.  on the other hand we should have the option to pin it so that if something does break users should be able pin the versions. but if we do that we have to update this for every release and I don't think that we have the bandwidth to take that on.  IDK, if someone has strong opinions one way or the other i could be swayed

Reply:
Any thoughts on a `stable` tag? Bumping it would be a change in Git rather than a change in this file

Reply:
tags are unique, so you would have to delete the old ones and create a new one when you feel its stable, which puts us back in the same position

Reply:
I believe you can force-push a tag to a new commit.  
The difference from this is that it's a decoupled process from this particular piece of code. 
The team deciding that a specific commit of SLOCTL is `Stable` can just be consumed by everyone else looking for almost-latest, but that team doesn't have to remember to update this particular file, or any other tools consuming that `Stable` version.

Reply:
you can force push them, but just a hacky version of what I was saying above.  My point is broadly about making another step in the process, so if that's updating things here or in the release process by making a tag, its not a difference that makes a distinction.  For now, I'm going to keep it as is